lib/options.json

```diff
@@ -306,6 +284,34 @@
       "description": "Specifies client properties. https://webpack.js.org/configuration/dev-server/#devserverclient",
       "additionalProperties": false
     },
+    "webSocketURL": {
```

Comment:
It should be in the `client` options, because this option only for client

Reply:
yep maybe I was confused before, fixed now 👍